e2e/atlas/online_archives_test.go

```diff
@@ -28,8 +28,6 @@ import (
 )
 
 func TestOnlineArchives(t *testing.T) {
-	const onlineArchiveEntity = "onlineArchives"
```

Comment:
this file is mostly refactor truing to test pause/start but I could not do that

Reply:
What was the issue?

Reply:
oh I have just seen your comment on the test doc

Reply:
We can't pause an online archive that's not active, and since this clusters are empty the only archive does nothing